overhaul/lifeboats.md

```diff
@@ -0,0 +1,68 @@
+## Abstract
+
+Lifeboats are a round-end system used during evacuation on hijack. 'Lifeboats' refer to the two large escape pods located on the aft upper deck of the Almayer.
+
+During the hijack sequence, as soon as the hijacked dropship crashes into the Almayer, a 10-minute countdown is automatically started and lifeboats begin the countdown process.
+
+Once the 10 minutes are up, lifeboats try to launch. If xenos are onboard either lifeboat when the launch begins, the lifeboats fail to launch.
+
+## Goals
+
+1. Lifeboats start the evacuation process immediately upon the dropship crashing
+2. Automate self-destruct and distress beacon sequences
+3. Distress beacon should be launched 2 minutes after crash
+4. Lifeboats should launch automatically once 12 minutes have passed
+5. BOTH lifeboats should fail if a xeno or facehugger is present on EITHER lifeboat at launch time
+6. Make them just fail to launch, don’t blow them up that’s silly
+7. Implement cascading emergency floor lighting directing people towards the lifeboats area
+8. Port over the fancy screens and timer from the original lifeboat PR
+9. Add self-destruct room to lower engineering. SD rods will self-insert themselves automatically
+10. SD rods need to become their own dedicated structure rather than being a wacky overlay
+11. The non-hijacked dropship (the one sitting pretty on the pad) should launch to perpetual flyby as its own escape pod
+12. Remove larva surge! It was meant to be a stopgap until lifeboats anyway.
+
+## Non-Goals
+
+1. Do not use tgshuttles for escape pods (yet). That's another rabbithole for another PR to handle.
+2. Do not overcomplicate this PR. It should do what is outlined above and avoid adding unnecessary complexity.

```

Comment:
Non-goals is the design for what you don't want the feature to be.
Do not include implementation detail like 'tgshuttles'

Reply:
Examples for this section:

## Non-Goals

1. Be a free escape for marines. Marines should have to fight to escape on the lifeboats, it should not be an easy escape.